File: app/java/fas/src/main/java/com/example/accounting/application/exception/AccountNotFoundException.java
package com.example.accounting.application.exception;

/**
 * 勘定科目が見つからない例外
 */
public class AccountNotFoundException extends RuntimeException {

    public AccountNotFoundException(String message) {
        super(message);
    }

    public AccountNotFoundException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: app/csharp/AccountingSystem.Application/Exceptions/AccountNotFoundException.cs
namespace AccountingSystem.Application.Exceptions;

/// <summary>
/// 勘定科目が見つからない例外
/// </summary>
public class AccountNotFoundException : Exception
{
    public AccountNotFoundException(string message) : base(message)
    {
    }

    public AccountNotFoundException(string message, Exception innerException) : base(message, innerException)
    {
    }
}
